### ISSUE TITLE
bug: Error When Starting aw-watcher.nvim

### ISSUE THREAD

**Issue by try-to-fly** [lowitea/aw-watcher.nvim#1]
### Did you check docs and existing issues?

- [X] I have read all the plugin docs
- [X] I have searched the existing issues
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

v0.10.2

### Operating system/version

MacOs 15.2

### Describe the bug

I followed the instructions in the README to install the aw-watcher.nvim plugin, but I encountered an error when starting.

```
Failed to run `config` for aw-watcher.nvim

...hare/nvim/lazy/aw-watcher.nvim/lua/aw_watcher/config.lua:34: bucket hostname must be a string

# stacktrace:
  - /aw-watcher.nvim/lua/aw_watcher/config.lua:34 _in_ **new**
  - /aw-watcher.nvim/lua/aw_watcher/init.lua:57 _in_ **setup**
  - lua/config/lazy.lua:9
  - init.lua:2
```

### Steps To Reproduce

-

### Expected Behavior

-

**Comment by lowitea** [comment]
Hi!
Thank you. I don't have a MacOs to test, so I'm very grateful that you opened the issue :)
I hope I fixed the issue with the 16bc4a7 commit.

**Comment by try-to-fly** [comment]
The new code works fine. Thank you!
